configs/stac/stac_fly_treadmill.yaml

```diff
@@ -0,0 +1,15 @@
+fit_path: "fly_treadmill_fit.p"
+transform_path: "transform_treadmill.p"
+# File is too large to commit
+# DL from: https://datadryad.org/stash/dataset/doi:10.5061/dryad.mpg4f4r73
+# Actual file: https://datadryad.org/stash/downloads/file_stream/3361804
+data_path: "/data/users/eabe/biomech_model/Flybody/datasets/Tuthill_data/wt_berlin_linear_treadmill_dataset.csv"
```

Comment:
maybe change this path to just `wt_berlin_linear_treadmill_dataset.csv`

Reply:
It's a good idea, but the file is too big to check in, so I am leaving this as is for now. For now, the comment indicates how to get the file. I should probably change the path prefix to point to our data dir. though.  But def. need to get a smaller file for unit testing etc. asap.